### PR TITLE
create docker compose service

### PR DESCRIPTION
**Pull request by lumjjb**
This PR creates a docker compose container that will run neo4j, nats and the ingestor.

You can run this via `make service` command (and `make container` command to create the necessary containers to run the service). A run would look like this, and after the ingestor is up, a collector can be run (e.g. ` bin/collector files /path/to/guac-data/docs`)

```
$ make service
# requires force recreate since docker compose reuses containers and neo4j does
# not handle that well.
#
# if container images are missing, run `make container` first
docker compose up --force-recreate
[+] Running 4/3
 ⠿ Container guac-neo4j-1           Recreated                                                                                                                                 0.1s
 ⠿ Container guac-nats-1            Recreated                                                                                                                                 0.1s
 ⠿ Container guac-service-health-1  Recreated                                                                                                                                 0.1s
 ⠿ Container guac-guac-ingestor-1   Recreated                                                                                                                                 0.1s
Attaching to guac-guac-ingestor-1, guac-nats-1, guac-neo4j-1, guac-service-health-1
guac-service-health-1  | checking-for-services
guac-nats-1            | [1] 2023/02/10 17:41:51.255056 [INF] Starting nats-server
guac-nats-1            | [1] 2023/02/10 17:41:51.255317 [INF]   Version:  2.9.14
guac-nats-1            | [1] 2023/02/10 17:41:51.255351 [INF]   Git:      [74ae59a]
guac-nats-1            | [1] 2023/02/10 17:41:51.255402 [INF]   Name:     NAKXBUWF35FN3VRCMZLG2MCUWIPJLTAIPHS7ZNXDWUGAJACVEBU4E7XT
guac-nats-1            | [1] 2023/02/10 17:41:51.255410 [INF]   Node:     wBj6ffLM
guac-nats-1            | [1] 2023/02/10 17:41:51.255413 [INF]   ID:       NAKXBUWF35FN3VRCMZLG2MCUWIPJLTAIPHS7ZNXDWUGAJACVEBU4E7XT
guac-nats-1            | [1] 2023/02/10 17:41:51.255474 [INF] Using configuration file: /config/nats/js.conf
guac-nats-1            | [1] 2023/02/10 17:41:51.255567 [WRN] Maximum payloads over 8.00 MB are generally discouraged and could lead to poor performance
guac-nats-1            | [1] 2023/02/10 17:41:51.257192 [INF] Starting http monitor on 0.0.0.0:8222
guac-nats-1            | [1] 2023/02/10 17:41:51.257330 [INF] Starting JetStream
guac-nats-1            | [1] 2023/02/10 17:41:51.257851 [INF]     _ ___ _____ ___ _____ ___ ___   _   __  __
guac-nats-1            | [1] 2023/02/10 17:41:51.257859 [INF]  _ | | __|_   _/ __|_   _| _ \ __| /_\ |  \/  |
guac-nats-1            | [1] 2023/02/10 17:41:51.257861 [INF] | || | _|  | | \__ \ | | |   / _| / _ \| |\/| |
guac-nats-1            | [1] 2023/02/10 17:41:51.257863 [INF]  \__/|___| |_| |___/ |_| |_|_\___/_/ \_\_|  |_|
guac-nats-1            | [1] 2023/02/10 17:41:51.257864 [INF]
guac-nats-1            | [1] 2023/02/10 17:41:51.257866 [INF]          https://docs.nats.io/jetstream
guac-nats-1            | [1] 2023/02/10 17:41:51.257868 [INF]
guac-nats-1            | [1] 2023/02/10 17:41:51.257870 [INF] ---------------- JETSTREAM ----------------
guac-nats-1            | [1] 2023/02/10 17:41:51.257875 [INF]   Max Memory:      5.76 GB
guac-nats-1            | [1] 2023/02/10 17:41:51.257878 [INF]   Max Storage:     12.04 GB
guac-nats-1            | [1] 2023/02/10 17:41:51.257881 [INF]   Store Directory: "/tmp/nats/jetstream"
guac-nats-1            | [1] 2023/02/10 17:41:51.257882 [INF] -------------------------------------------
guac-nats-1            | [1] 2023/02/10 17:41:51.258249 [INF] Listening for client connections on 0.0.0.0:4222
guac-nats-1            | [1] 2023/02/10 17:41:51.258562 [INF] Server is ready
guac-neo4j-1           | Changed password for user 'neo4j'. IMPORTANT: this change will only take effect if performed before the database is started for the first time.
guac-neo4j-1           | Fetching versions.json for Plugin 'apoc' from https://neo4j-contrib.github.io/neo4j-apoc-procedures/versions.json
guac-neo4j-1           | Installing Plugin 'apoc' from https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/4.4.0.9/apoc-4.4.0.9-all.jar to /var/lib/neo4j/plugins/apoc.jar
guac-neo4j-1           | Applying default values for plugin apoc to neo4j.conf
guac-service-health-1  | nats-up
guac-neo4j-1           | 2023-02-10 17:41:57.661+0000 INFO  Starting...
guac-neo4j-1           | 2023-02-10 17:41:58.052+0000 INFO  This instance is ServerId{8037b4e3} (8037b4e3-192b-46f3-99f2-e548c08e4501)
guac-neo4j-1           | 2023-02-10 17:41:59.247+0000 INFO  ======== Neo4j 4.4.9 ========
guac-neo4j-1           | 2023-02-10 17:42:04.612+0000 INFO  Performing postInitialization step for component 'security-users' with version 3 and status CURRENT
guac-neo4j-1           | 2023-02-10 17:42:04.612+0000 INFO  Updating the initial password in component 'security-users'
guac-neo4j-1           | 2023-02-10 17:42:06.090+0000 INFO  Called db.clearQueryCaches(): Query caches successfully cleared of 1 queries.
guac-neo4j-1           | 2023-02-10 17:42:06.148+0000 INFO  Bolt enabled on 0.0.0.0:7687.
guac-neo4j-1           | 2023-02-10 17:42:06.897+0000 INFO  Remote interface available at http://localhost:7474/
guac-neo4j-1           | 2023-02-10 17:42:06.901+0000 INFO  id: 030249234A8796D3DA4A72F6123CA534136F78D66D33A9C029D86CEA4F32FD01
guac-neo4j-1           | 2023-02-10 17:42:06.901+0000 INFO  name: system
guac-neo4j-1           | 2023-02-10 17:42:06.901+0000 INFO  creationDate: 2023-02-10T14:14:55.343Z
guac-neo4j-1           | 2023-02-10 17:42:06.901+0000 INFO  Started.
guac-service-health-1  | neo4j-up
guac-service-health-1 exited with code 0
guac-guac-ingestor-1   | {"level":"info","ts":1676050932.041539,"caller":"cmd/root.go:89","msg":"Using config file: /guac/guac.yaml"}
guac-guac-ingestor-1   | {"level":"info","ts":1676050932.0459363,"caller":"emitter/nats_emitter.go:121","msg":"creating stream \"DOCUMENTS\" and subjects \"DOCUMENTS.*\""}
guac-guac-ingestor-1   | {"level":"info","ts":1676050937.787074,"caller":"emitter/nats_emitter.go:193","msg":"[ingestor: df3a6c7f-65a6-4e0e-86a6-487dc0862859] nothing to consume, backing off for 1s: %!w(*errors.errorString=&{nats: timeout})"}
guac-guac-ingestor-1   | {"level":"info","ts":1676050937.788511,"caller":"emitter/nats_emitter.go:193","msg":"[processor: 76841ccf-ecf4-4f60-bcca-52c60479cf76] nothing to consume, backing off for 1s: %!w(*errors.errorString=&{nats: timeout})"}
```

Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

File: container_files/nats/js.conf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// change max payload from default 1MB to 8MB
```
Comment (pxp928) [inline container_files/nats/js.conf:1]: the comment here needs to change to 64MB for consistency (it was wrong when I sent it to you)

**Review by pxp928 — APPROVED** [review]
LGTM just a small comment change.

File: container_files/nats/js.conf (hunk 1 above)
Comment (lumjjb) [reply container_files/nats/js.conf]: done!

File: dockerfiles/Dockerfile.healthcheck [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM ubuntu:20.04
```
Comment (mlieberman85) [inline dockerfiles/Dockerfile.healthcheck:1]: Nit: the above one is ubuntu:22.04 this one is 20.04

Comment (lumjjb) [reply dockerfiles/Dockerfile.healthcheck]: fixed! tks

**Review by mlieberman85 — APPROVED** [review]
LGTM

**Pull request merged**